migration/actual_migrations_test.go

```diff
@@ -1083,3 +1083,116 @@ func TestMigration28(t *testing.T) {
 	// not possible to insert more than 1 row per org
 	assert.Error(t, err)
 }
+
+func TestMigration29(t *testing.T) {
+	db, dbDriver, closer := prepareDBAndInfo(t)
+	defer closer()
+
+	if dbDriver == types.DBDriverSQLite3 {
+		// nothing worth testing for sqlite
```

Comment:
;-)

Reply:
well it's true, sqlite wouldn't be able to handle these migrations.. should I change it to the usual `sqlite is not supported anymore` or just start ignoring it completely?